README.md

```diff
@@ -30,6 +30,10 @@ The following images are currently published and updated by the distroless proje
 
 | Image                                 | Tags                                   | Architecture Suffixes             | 
 | ---                                   | ---                                    | ---                               |
+| gcr.io/distroless/static-debian12     | latest, nonroot, debug, debug-nonroot  | amd64, arm64, arm, s390x, ppc64le |
+| gcr.io/distroless/base-debian12       | latest, nonroot, debug, debug-nonroot  | amd64, arm64, arm, s390x, ppc64le |
+| gcr.io/distroless/base-nossl-debian12 | latest, nonroot, debug, debug-nonroot  | amd64, arm64, arm, s390x, ppc64le |
+| gcr.io/distroless/cc-debian12         | latest, nonroot, debug, debug-nonroot  | amd64, arm64, arm, s390x, ppc64le |
```

Comment:
These actually do not hold true with the current configuration of the BUILD. There are a few more modifications there needed to make a claim that we are building all the variants.

Reply:
I'll try to find some time to address those today. Also, I've spent some time to try to fix the broken test but I couldn't figure why it fails. Any clue?